Jenkinsfile

```diff
@@ -11,7 +11,7 @@ app.build(
     requestCpu: '1',
     limitCpu: '2',
     requestMemory: '6Gi',
-    limitMemory: '12Gi',
+    limitMemory: '8Gi',
```

Comment:
Does the compilation fit into this much memory? Unlike CPU, exceeding this will cause your builds to be aborted, not to run slower.

Reply:
Turns out no, it doesn't fit. This PR's CI build was killed because of OOM. Charts in [HQ](https://grafana-hq.powerapp.cloud/d/hYTBQhQiz/pod-resource-usage?orgId=1&refresh=10s&from=now-2d&to=now&var-namespace=jenkins-builds&var-filter=jenkins-powerhome-playbook&var-pod=All) and [Gotham](https://grafana-gm.powerapp.cloud/d/hYTBQhQiz/pod-resource-usage?orgId=1&refresh=10s&from=now-2d&to=now&var-namespace=jenkins-builds&var-filter=jenkins-powerhome-playbook&var-pod=All) show that memory utilisation generally reaches up to around 11.5Gi, so if anything you'll need to increase this, not reduce it.

Of course, making other fixes that reduce the actual memory consumption would be great, but you'd have to do that and watch that consumption fall on those charts before you could lower the limit.

Reply:
Good to know. I will back this one out.